shared/domains/gi/db.js

```diff
@@ -37,9 +37,15 @@ export default (sbp('sbp/selectors/register', {
   'gi.db/set': function (key: string, value: string): Promise<void> {
     return Promise.resolve(sbp('okTurtles.data/set', key, value))
   },
+  'gi.db/delete': function (key: string) {
```

Comment:
Shouldn't there be a `Promise<void>` return type annotation here like for 'gi.db/set'?

Reply:
I fixed this. It'll be in the next commit